viewShip.php

```diff
@@ -0,0 +1,52 @@
+<?php
+require __DIR__.'/bootstrap.php';
+// page to display a single ship data with the options to:
+// ADD, EDIT, DELETE
+use Service\Container;
+
+
+
+$id = $_GET['id'];
+
+// Need shiploader obj for ship
+$container = new Container($configuration);
+$shipLoader = $container->getShipLoader();
+$ship = $shipLoader->findOneById($id); // values may be getting changed here
+
+require 'layout/header.php';
+
+?>
+
+    <div class="container">
+            <div class="page-header">
+                <h1>Viewing the <?php echo $ship->getName()?></h1>
+            </div>
+            <a href="/manageShips.php" class="btn btn-md btn-primary pull-right">Manage Ships</a>
+            
+            <table class="table table-striped">
+                <tbody>
+                    <tr class="d-flex">
```

Comment:
Are you using the `d-flex` and `col-1` here to make the first column smaller? If so, I think an easier approach would be to add something like `style="width: 100px;"` to the first `th` element. The others will follow the same width.